infra/core/monitor/loganalytics.bicep

```diff
@@ -0,0 +1,22 @@
+metadata description = 'Creates a Log Analytics workspace.'
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
+resource logAnalytics 'Microsoft.OperationalInsights/workspaces@2021-12-01-preview' = {
```

Comment:
should this be preview?

Reply:
@jongio - I checked the files in the latest core and it is still this version. Do we need to change it?

Reply:
Yes, there are features in that preview that are needed.  We can dig in more if needed.